parts/k8s/cloud-init/artifacts/cse_main.sh

```diff
@@ -224,23 +201,23 @@ if $FULL_INSTALL_REQUIRED; then
     fi
 fi
 
-{{if not IsAzureStackCloud}}
-if [[ $OS == $UBUNTU_OS_NAME ]] && [[ "${TARGET_ENVIRONMENT,,}" != "${AZURE_STACK_ENV}"  ]]; then
+{{- if not IsAzureStackCloud}}
+if [[ $OS == $UBUNTU_OS_NAME ]]; then
     apt_get_purge 20 30 120 apache2-utils &
 fi
 {{end}}
 
 if $REBOOTREQUIRED; then
-  echo 'reboot required, rebooting node in 1 minute'
-  /bin/bash -c "shutdown -r 1 &"
-  if [[ $OS == $UBUNTU_OS_NAME ]]; then
-      aptmarkWALinuxAgent unhold &
-  fi
+    echo 'reboot required, rebooting node in 1 minute'
```

Comment:
I prefer 2-space indentation in shell scripts (and I think that's more common in our codebase), but ideally we would agree on a standard. (See #2297.)

Reply:
My IDE was detecting that the large majority of indentation was 4 whitespace, so I applied that standard uniformly.

Happy to schedule a follow-up thread to discuss an abstract standard, and then we can apply that standard to shell scripts generally.